docs/app/index.ejs

```diff
@@ -44,7 +44,7 @@
       function removeDimmer() {
         dimmer.style.opacity = '0'
 
-        setTimeout(() => {
+        setTimeout(function() {
```

Comment:
[Arrow functions](http://caniuse.com/#feat=arrow-functions) are not supported by IE.